client/src/header/components/Header.tsx

```diff
@@ -1,16 +1,17 @@
 import React, { FC } from "react";
 import { PageHeader, Button, Menu, Dropdown, Icon, Tag } from "antd";
-import { Link } from "react-router-dom";
+import { Link, Redirect } from "react-router-dom";
 
 interface Props {
   isLoggedIn: boolean;
   name?: string;
+  receivedError: boolean;
   onLogin: () => void;
   onLogout: () => void;
 }
 
 // TODO(Ryan): Should add a isLoggedIn prop, if they are then display user's name
-const Header: FC<Props> = ({ onLogin, isLoggedIn, name, onLogout }) => {
+const Header: FC<Props> = ({ onLogin, isLoggedIn, receivedError, name, onLogout }) => {
```

Comment:
This should definitely live in the canvas component. Since `Header` is present on all pages, we could end up with an infinite loop if we put this here...